content/blog/2020-06-29-optimizing-dvc-remotes.md

```diff
@@ -0,0 +1,174 @@
+---
+title: Optimizing DVC Remotes
+date: 2020-06-29
+description: |
+  An overview of how syncing data to and from remote storage is optimized in DVC.
+picture: 2020-05-04/owl.png
+author: peter_rowlands
+---
+
+One of the key features provided by DVC is the ability to efficiently sync
+versioned datasets between a user's local machine and
+[remote storage](https://dvc.org/doc/command-reference/remote), and version 1.0
+includes several performance optimizations related to syncing data with remotes.
```

Comment:
I would start if possible with something like "Our users have presented the need for optimizing remotes blah blah" and give some examples e.g. Discord message screenshots.

Reply:
Also, minor: I personally prefer "synchronizing" or "syncing". The pronunciation of the latter is questionable, no?